extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -557,6 +559,11 @@ export class AjaxMonitor extends BaseTelemetryPlugin implements IDependenciesPlu
                 }
             }
 
+            if (_self._config.includeResponseErrorData && xhr.status !== 200) {
+                dependency.properties = dependency.properties || {};
+                dependency.properties.responseError = xhr.statusText + " - " + xhr.responseText;
```

Comment:
nit: rename to `responseMessage` or `responseText`, to convey string type instead of Error.